fedimintd/src/fedimintd.rs

```diff
@@ -128,8 +131,11 @@ fn parse_map(s: &str) -> anyhow::Result<BTreeMap<String, String>> {
 /// }
 /// ```
 pub struct Fedimintd {
+    /// Registry for initializing server modules
```

Comment:
"Registry of available module kinds". 

Module Init is kind of a constructor for a module (Fedimint could possibly construct multiple module of the same kind) - with bunch of function to handle module before it's really initialized.